lightyear/src/utils/avian3d.rs

```diff
@@ -70,13 +71,19 @@ pub mod position {
     }
 }
 
+/// copied from Animatable trait in bevy_animation, which we don't want as a dep because
```

Comment:
The Lerp traits come from Animatable?
Maybe now there's way to get them from the new curve API in a headless manner

Reply:
> The Lerp traits come from Animatable? Maybe now there's way to get them from the new curve API in a headless manner

yes (although all it does is call `slerp`) i've not checked yet, but it could be that bevy 0.15 has something suitable in the new curve api.